### PR TITLE
Feat: Emailjs Setup

### LINKED ISSUE #5: Setup Email.js
## Summary

Email.js will be used for the contact us page. Set up that and test it so that we can just call a function later on when we set up the contact us page. Make the function live in the `src/services/contact.js`

### Screenshots
N/A

### Basic example
N/A

### Motivation

So that users can send the team an email.

### PR DESCRIPTION
**Pull request by nasamsay**
create handleSubmit function to call on contact us page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,10 +7,12 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.0.1",
     "@testing-library/user-event": "^14.1.1",
+    "emailjs-com": "^3.2.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.1",
     "react-scripts": "5.0.1",
+    "sweetalert2": "^11.6.2",
```
Comment (MuhannedNoman) [inline package.json]: We have an alert in tailwind, so I will be creating an issue to create an alert component. So you can remove this.

File: src/services/contact.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import emailjs from 'emailjs-com';
+import Swal from 'sweetalert2';
+
+const SERVICE_ID = 'service_3tixi8d';
+const TEMPLATE_ID = 'template_tsw4ro5';
+const USER_ID = 'Cprut94nRZj80oKe-';
+
+const handleSubmit = (e) => {
+    e.preventDefault();
+    emailjs.sendForm(SERVICE_ID, TEMPLATE_ID, form.current, USER_ID)
```
Comment (MuhannedNoman) [inline src/services/contact.js]: Where does the form variable come from?

File: src/services/contact.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      .then((result) => {
+        console.log(result.text);
+        Swal.fire({
+          icon: 'success',
+          title: 'Message Sent Successfully'
+        })
```
Comment (MuhannedNoman) [inline src/services/contact.js]: Add a TODO item here and we can replace this latter with the Alert.

File: src/services/contact.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      }, (error) => {
+        console.log(error.text);
+        Swal.fire({
+          icon: 'error',
+          title: 'Ooops, something went wrong',
+          text: error.text,
```
Comment (MuhannedNoman) [inline src/services/contact.js]: Same as above

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
LGTM, but I have requested some changes. Let me know what you think.

File: src/services/contact.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import emailjs from 'emailjs-com';
+
+const SERVICE_ID = 'service_3tixi8d';
+const TEMPLATE_ID = 'template_tsw4ro5';
+const USER_ID = 'Cprut94nRZj80oKe-';
+
+const handleSubmit = (text) => {
```
Comment (MuhannedNoman) [inline src/services/contact.js]:
```suggestion
const senEmail= (text) => {
```

File: src/services/contact.js [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    emailjs.sendForm(SERVICE_ID, TEMPLATE_ID, text, USER_ID)
+      .then((result) => {
+        //TODO: create alert if its successful
+      }, (error) => {
+        //TODO: create alert if something went wrong
+      });
+}
+
+export default handleSubmit;
```
Comment (MuhannedNoman) [inline src/services/contact.js]:
```suggestion
export default sendEmail;
```

**Review by MuhannedNoman — DISMISSED** [review]
LGTM, thank you.

**Review by hayatzsala — DISMISSED** [review]

File: src/services/contact.js [hunk 7]
```diff
@@ -0,0 +1,16 @@
+import emailjs from 'emailjs-com';
+
+const SERVICE_ID = 'service_3tixi8d';
+const TEMPLATE_ID = 'template_tsw4ro5';
+const USER_ID = 'Cprut94nRZj80oKe-';
+
+const senEmail= (text) => {
```
Comment (MuhannedNoman) [inline src/services/contact.js]:
```suggestion
const sendEmail= (text) => {
```

**Review by MuhannedNoman — DISMISSED** [review]

**Review by MuhannedNoman — APPROVED** [review]

**Pull request merged**